assets/js/modules/analytics/setup.js

```diff
@@ -656,12 +664,24 @@ class AnalyticsSetup extends Component {
 						</Radio>
 					</Fragment>
 				}
+				{ onSettingsPage && useSnippet && (
+					<div className="googlesitekit-setup-module__input">
+						<Switch
+							id="anonymizeIPAddress"
+							label={ __( 'Anonymize IP address', 'google-site-kit' ) }
```

Comment:
Should this maybe say "Anonymize IP addresses" (plural) instead of singular? This would be more consistent with the description as well.

Reply:
Yeah, that's fair. 👍